test/application_system_test_case.rb

```diff
@@ -22,5 +22,9 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
     Capybara::Selenium::Driver.new(app, browser: :chrome, options: options)
   end
 
-  driven_by driver, using: :chrome, screen_size: [1400, 1400]
+  begin
+    driven_by driver, using: :chrome, screen_size: [1400, 1400]
+  rescue Webdrivers::BrowserNotFound
+    driven_by :selenium, using: :firefox, screen_size: [1400, 1400]
```

Comment:
should we not add this in for now since the tests don't work in firefox yet?

Reply:
My thought on this is it's for local testing. Some things MAY work... maybe we put in a comment. We still only use chrome in CI.

Reply:
that makes sense!